manual/manual_03.asciidoc

```diff
@@ -1673,14 +1673,14 @@ The predefined `opaque` attribute is only available in instances and classes hav
 
 When an instance with the `Container` property is encountered during game play, it will be described as usual.
 By default the contents of the container will be listed unless it is empty or opaque.
-If the default description has been overridden the author will have to explicitly decide on how to handle the container aspect and its content.
+If the default description has been overridden, the author will have to explicitly decide on how to handle the container aspect and its content.
 
 
 
 ==== LIMITS
 
-The (((LIMITS, clause))) `Limits` clause of the `Container` property declaration makes it possible to put limitations on what and how much can be put in the container.
-Those limitations are enforced by the interpreter during gameplay.
+The (((LIMITS, clause))) `Limits` clause of the `Container` property declaration makes it possible to enforce limitations on _what_ and _how much_ can be put in any given container (as a class or a instance).
```

Comment:
I don't like the word "enforce" when it comes to what the author does. He only defines, so maybe that is a better word. The enforcing of those limits is done by the interpreter.